internal/cli/command/cluster/feature/features/dns/activate.go

```diff
@@ -438,6 +479,8 @@ func getFeatureSpecDefaults(banzaiCLI cli.Cli, provider Provider) (DNSFeatureSpe
 			return DNSFeatureSpec{}, errors.WrapIf(err, "failed to parse DNS capabilities")
 		}
 
+		clusterDomain := fmt.Sprintf("%s.%s", clusterCtx.ClusterName(), dnsCapability.BaseDomain)
```

Comment:
Shouldn't the organization name also be added to the cluster domain?

Reply:
nice, thx